src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -602,6 +602,64 @@ static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization
  */
 static constexpr Property<element::Type, PropertyMutability::RW> kv_cache_precision{"KV_CACHE_PRECISION"};
 
+/**
+ * @brief Enum to define possible cache quant schema hints.
+ */
+enum class CacheQuantMode {
+    AUTO = 0,
+    BY_CHANNEL = 1,
+    BY_TOKEN = 2
+};
+
+/** @cond INTERNAL */
+inline std::ostream& operator<<(
+    std::ostream& os, const CacheQuantMode& mode) {
+    switch (mode) {
+        case CacheQuantMode::AUTO:
+            return os << "AUTO";
+        case CacheQuantMode::BY_CHANNEL:
+            return os << "BY_CHANNEL";
+        case CacheQuantMode::BY_TOKEN:
+            return os << "BY_TOKEN";
+        default: OPENVINO_THROW("Unsupported cache quant mode");
+    }
+}
+/** @endcond */
+
+inline std::istream& operator>>(
+    std::istream& is, CacheQuantMode& mode) {
+    std::string str;
+    is >> str;
+    if (str == "AUTO") {
+        mode = CacheQuantMode::AUTO;
+    } else if (str == "BY_CHANNEL") {
+        mode = CacheQuantMode::BY_CHANNEL;
+    } else if (str == "BY_TOKEN") {
+        mode = CacheQuantMode::BY_TOKEN;
+    } else {
+        OPENVINO_THROW("Unsupported cache quant mode: ", str);
+    }
+    return is;
+}
+
+/**
+ * @brief Define quant mode for key cache.
+ * @param AUTO - default mode by plugin
+ * @param BY_CHANNEL - quant by channel
+ * @param BY_TOKEN - quant by token
+ */
+
+static constexpr Property<CacheQuantMode, PropertyMutability::RW> key_cache_quant_mode{"KEY_CACHE_QUANT_MODE"};
```

Comment:
These properties where internal.
They can be common but still internal.

Reply:
Yes to move to internal properties check file:
`src/inference/dev_api/openvino/runtime/internal_properties.hpp`
 which is part of dev API, it will be a common to be used by plugins but not available for external users.
As these was internal before and not exposed it shouldn't be exposed because they are common

Reply:
talked offline, will move it to internal_properties.hpp